<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_move">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Перенос</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_dx">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>41</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="maxLength">
          <number>32767</number>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_dy">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>50</y>
           <width>41</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_dz">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>41</width>
           <height>24</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_dx">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>dX</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_dy">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>dY</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_dz">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>30</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>dZ</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_move">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Перенести</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_scale">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>280</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Масштабирование</string>
        </property>
        <widget class="QGroupBox" name="groupBox_coords">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>171</width>
           <height>81</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Коэффициенты</string>
         </property>
         <widget class="QLineEdit" name="lineEdit_KZ">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_KY">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>KY</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_KX">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>KX</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_KY">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_KZ">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>30</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>KZ</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_KX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="button_scale">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Масштабировать</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_rotation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>280</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>280</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Поворот</string>
        </property>
        <widget class="QGroupBox" name="groupBox_degree">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>161</width>
           <height>81</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Угол поворота</string>
         </property>
         <widget class="QLabel" name="label_dgZ">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>30</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>dgZ</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_dgX">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_dgY">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_dgX">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>dgX</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_dgY">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>dgY</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_dgZ">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="button_rotate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Повернуть</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_ObjManage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>360</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLineEdit" name="lineEdit_file">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>50</y>
             <width>181</width>
             <height>24</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_file">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>25</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Имя файла</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_load">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_unload">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>80</y>
             <width>151</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Выгрузить</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_graphics">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string> Экран</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_move</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_scale</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_rotate</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>668</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_unload</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_move</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_unload</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_scale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_unload</sender>
   <signal>clicked(bool)</signal>
   <receiver>button_rotate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>675</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_unload</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_load</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_unload</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_unload</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_unload</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_load</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
